### DISCUSSION TITLE
Error When Connecting Multiple Sheets

### DISCUSSION THREAD

**Discussion by MarcusSorealheis** [Budibase/budibase#14698] [Help]
Hello there,

I recently went to connect an app to a Google Sheets "backend," but there was an error consistently fetching the sheets:

<img width="633" alt="image" src="https://github.com/user-attachments/assets/0a5b94bc-774b-4af3-bd98-08c2c41f56fb">

I'm using hosted Budibase and I have multi-factor authentication for my Google account in case those are helpful details.

**Comment by therealkamil** [comment]
+1 Same issue here

**Comment by feurer98** [comment]
+1 Same here

Was there a solution here?

**Comment by Alfie-level** [comment]
+1 Same Here 

I write email to support team, still waiting for feedback.
